classification/classification_opvn.py

```diff
@@ -0,0 +1,39 @@
+import tensorflow as tf
+import numpy as np
+import cv2 as cv
```

Comment:
There is no much difference between `classification_tf` and `classification_opvn` - please create a single `classification.py` which accepts flag for target engine